src/sentry/middleware/stats.py

```diff
@@ -16,11 +18,11 @@ def add_request_metric_tags(request, **kwargs):
 
 
 class ResponseCodeMiddleware(MiddlewareMixin):
-    def process_response(self, request, response):
+    def process_response(self, request: Request, response):
```

Comment:
This is missing the `Response` annotations (happens in some other files where `process_response` is defined, too).